rep-0003.rst

```diff
@@ -233,6 +233,66 @@ Buildsystem support:
 
   - build from source
 
+Lunar Loggerhead (May 2017 - May 2019)
+----------------------------------
+Required Support for:
+
+- Ubuntu Xenial (16.04)
+- Ubuntu Yakkety (16.10)
+- Ubuntu Zesty (17.04)
+
+Recommended Support for:
+
+- Debian Stretch
+- Fedora 25
+
+Architecture supported(CI enabled for):
+- amd64
+- arm32
+- arm64
+Not building x86 packages anymore
```

Comment:
Also you may want to render the page to see what this section looks like. I think Markdown usually requires a blank line before and after the list to render it as a separate item.